test/es6/generators-apis.js

```diff
@@ -50,6 +50,39 @@ var tests = [
         }
     },
     {
+        // Regression Test GitHub Issue #3065: Generator length property is inconsistently configurable (+ assert failure in debug build) 

```

Comment:
There is good coverage for built-in properties like caller in the other tests in this file. I will add a test case for non-built-in property.

---
In reply to: [128838455](https://github.com/Microsoft/ChakraCore/pull/3413#discussion_r128838455) [](ancestors = 128838455)